sql/core/src/test/resources/sql-tests/analyzer-results/charvarchar.sql.out

```diff
@@ -465,7 +465,7 @@ Project [ascii(c7#x) AS ascii(c7)#x, ascii(c8#x) AS ascii(c8)#x, ascii(v#x) AS a
 -- !query
 select base64(c7), base64(c8), base64(v), ascii(s) from char_tbl4
 -- !query analysis
-Project [base64(cast(c7#x as binary)) AS base64(c7)#x, base64(cast(c8#x as binary)) AS base64(c8)#x, base64(cast(v#x as binary)) AS base64(v)#x, ascii(s#x) AS ascii(s)#x]
+Project [base64(cast(c7#x as binary), false) AS base64(c7)#x, base64(cast(c8#x as binary), false) AS base64(c8)#x, base64(cast(v#x as binary), false) AS base64(v)#x, ascii(s#x) AS ascii(s)#x]
```

Comment:
ditto

Reply:
Do you mean to override `toString` method?